packages/react/src/table/TableScroller.tsx

```diff
@@ -209,11 +209,25 @@ export function TableScroller({ children }: TableScrollerProps) {
 	const hasScroll = thumbWidthRatio !== 1;
 
 	useEffect(() => {
-		setScrollerAriaLabel(
-			`Table ${
-				scrollerRef.current?.querySelector('caption')?.textContent || ''
-			}`
-		);
+		let ariaLabel = '';
+		const captionEl = scrollerRef.current?.querySelector('caption');
+
+		if (captionEl) {
+			ariaLabel = captionEl?.textContent || '';
```

Comment:
There are so many `|| ''` here because TS whinges at every step of the way ε(´סּ︵סּ`)з

Reply:
Line 212 tells TS that `ariaLabel` should only be a string. TS warning us that we're trying to set this value to something else seems beneficial since we're stringifying it on line 230 and could potentially end up announcing "Table undefined" if e.g. `undefined` was allowed. 

I think what you have is fine (except I don't fully understand the scroller potentially being called `Table `). Alternatively, you could remove the two `|| ''` and do something like this:

```ts
// Line 212
let ariaLabel: string | null | undefined;

// Line 230
setScrollerAriaLabel(`Table ${ariaLabel || ''}`);
```

Reply:
Ah, that's much better. Thanks